client/src/components/DataFiles/DataFilesListing/DataFilesListing.test.jsx

```diff
@@ -324,3 +324,99 @@ describe('DataFilesListing - Section Name Determination', () => {
     ).toBeInTheDocument();
   });
 });
+describe('DataFilesListing - showViewPath', () => {
```

Comment:
```suggestion
describe('DataFilesListing - showViewPath', () => {
  afterEach(() => {
    vi.restoreAllMocks();
  });
```

Reply:
Suggested change added in https://github.com/TACC/Core-Portal/pull/992/commits/d452a10c705cbdb13fba0e8eccd9266ba272d4aa